jsonschema/_validators.py

```diff
@@ -233,24 +288,35 @@ def maxLength(validator, mL, instance, schema):
         yield ValidationError("%r is too long" % (instance,))
 
 
-def dependencies(validator, dependencies, instance, schema):
+def dependentRequired(validator, dependentRequired, instance, schema):
+    """
+    Split from dependencies
```

Comment:
Since all functions are named after the spec in JSON Schema only a link to the Spec would make sense, but this is self-explanatory. So I removed the docstring.